Database/Corrections/cataQuestFixes.lua

```diff
@@ -1583,6 +1583,9 @@ function CataQuestFixes.Load()
         [14267] = { -- Investigating the Sea Shrine
             [questKeys.preQuestSingle] = {14258},
         },
+        [14268] = { -- Deep Impact
+            [questKeys.objectives] = {nil,{{195659,nil,Questie.ICON_TYPE_INTERACT}}},
```

Comment:
This is not the way to fix this, as the actual objective is still of type "item". We are using the `requiredSourceItems` table to fix this and add https://www.wowhead.com/cata/item=49102/ancient-tablet-fragment there. Leaving the objective as it is.

Reply:
https://github.com/Questie/Questie/pull/6217/commits/edd7b7f4269db10c7994e24d340c3d9ac8a646ba